src/StripeTests/Services/Subscriptions/StripeSubscriptionCreateOptionsTest.cs

```diff
@@ -0,0 +1,49 @@
+namespace StripeTests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+
+    using Stripe;
+    using Stripe.Infrastructure;
+    using Xunit;
+
+    public class StripeSubscriptionCreateOptionsTest : BaseStripeTest
+    {
+        private StripeSubscriptionService service;
+
+        public StripeSubscriptionCreateOptionsTest()
+        {
+            this.service = new StripeSubscriptionService();
+        }
+
+        [Fact]
+        public void Serialize()
+        {
+            var options = new StripeSubscriptionCreateOptions()
+            {
+                CustomerId = "cus_123",
+                Items = new List<StripeSubscriptionItemOption>
```

Comment:
minor but we should get better with those names. For a long time I thought `xxxOption` versus `xxxOptions` meant different approaches.